spec/system/admin/products_spec.rb

```diff
@@ -256,6 +256,81 @@
     end
   end
 
+  describe "deleting" do
+    let!(:product1) { create(:simple_product, name: 'a product to keep', supplier: @supplier) }
```

Comment:
Info for reviewers: While in the middle of this PR, I've realized, it is perhaps not such a good idea to remove tests from the legacy design just yet, as we still have it in production. Hence added this commit to revert those changes (apologies for the detour!)